src/App.js

```diff
@@ -52,25 +51,25 @@ class App extends React.Component {
       }
   }
 
-  updateData(position, maskStatus) {
+  updateData(position, iconStatus) {
 
     let body = {
       timestamp: new Date().toUTCString(),
-      mask_status: maskStatus,
+      icon_status: iconStatus,
```

Comment:
let's agree on a standard, either camelCase, or snakeCase. I would suggest camel case since that's the standard in React already